src/content/docs/fr/guides/imports.mdx

```diff
@@ -87,9 +109,7 @@ Astro prend en charge l'importation de fichiers JSON directement dans votre appl
 import './style.css';
 ```
 
-Astro supporte l'importation de fichiers CSS directement dans votre application. Les styles importés n'exposent aucune exportation, mais l'importation d'un style ajoutera automatiquement ces styles à la page. Cela fonctionne pour tous les fichiers CSS par défaut, et peut prendre en charge les langages de compilation vers le CSS comme Sass et Less via des plugins.
-
-Si vous préférez ne pas écrire de CSS, Astro prend également en charge toutes les bibliothèques CSS-in-JS populaires (ex : styled-components) pour le style.
+Astro prend en charge l'importation de fichiers CSS directement dans votre application. Les styles importés n'exposent à aucune exportation, mais l'importation d'un style ajoutera automatiquement ce style à la page. Cela fonctionne pour tous les fichiers CSS par défaut, et peut prendre en charge les langages CSS compilés comme Sass et Less via des plugins..
```

Comment:
I also have a problem with this sentence: `Les styles importés n'exposent à aucune exportation`. Maybe with `ne nécessitent pas d'exportation` that could be better.
But anyway, you duplicate the final dot `..`
```suggestion
Astro prend en charge l'importation de fichiers CSS directement dans votre application. Les styles importés ne nécessitent pas d'exportations, mais l'importation d'un style ajoutera automatiquement ce style à la page. Cela fonctionne pour tous les fichiers CSS par défaut, et peut prendre en charge les langages CSS compilés comme Sass et Less via des plugins.
```